UnityProject/Assets/Scripts/ScriptableObjects/RP/EmoteSO.cs

```diff
@@ -108,5 +148,28 @@ protected bool CheckHandState(GameObject player)
 		{
 			return !player.GetComponent<PlayerScript>().playerMove.IsCuffed && Validations.HasBothHands(player);
 		}
+
+		/// <summary>
+		/// Checks if the player is in critical condition or not
+		/// </summary>
+		/// <param name="player">The player that you want to check their health.</param>
+		/// <returns>True if crit, false otherwise.</returns>
+		protected bool CheckPlayerCritState(GameObject player)
```

Comment:
Would be nice if that there was a sanded function of this, but this is good for now